### ISSUE TITLE
Make a list of datatypes and data structures already implemented and ones being worked upon by the back-end team. 

### ISSUE THREAD

**Issue by Madhav2310** [lcompilers/lpython#486]
It would give the functions implementing team much higher clarity on the range of functions and modules that can be implemented at this stage. We can keep updating in this thread. There are multiple different issues on it already #460 #455 #395 and more...

**Comment by certik** [comment]
Yes, we should document the current status of LPython in terms of features and what works.

@Madhav2310 do you want to start such a document? You can create a wiki page, create a list of such features, and let's link issues for things that are planned, and put a "checkmark" to features that already work.

**Comment by Madhav2310** [comment]
Absolutely! Would love to work on it, shall I make a file in the repository itself or on some other webtool @certik?

**Comment by certik** [comment]
Are you able to create a wiki document here: https://github.com/lcompilers/lpython/wiki ? If not, I can give you permissions.

**Comment by Madhav2310** [comment]
I'm unable to right now, would need permissions.

**Comment by certik** [comment]
Try now.
